platform/darwin/src/MGLAccountManager.h

```diff
@@ -1,11 +1,12 @@
-#import <Foundation/Foundation.h>
```

Comment:
Apparently we need these import statements or else Swift declarations in the generated documentation lack specific types: realm/jazzy#609. Importing required system frameworks is good practice anyways, unless you use a precompiled header that imports them.

(Sorry, I started this review a couple weeks ago but forgot to publish it.)